transparency-log.md

```diff
@@ -0,0 +1,51 @@
+# US-RSE Code of Conduct Transparency Log
+
+The US-RSE expects everyone who participates in the community will honor our
+[Code of Conduct](https://us-rse.org/about/code-of-conduct/). This includes
+anyone participating in any US-RSE-related activities, anyone claiming
+affiliation with the US-RSE, and especially anyone representing the US-RSE
+Association in any role (including as a volunteer or speaker).
+
+This log serves to inform the US-RSE community about all responses to Code of
+Conduct reports in a public, aggregated, and anonymized form. We believe that a
+community must be able to see that a Code of Conduct is enforced in order for it
+to be effective. Without a visible process, potential community members may not
+feel comfortable joining. A visible log can also help the community establish
+the social standards expressed in the Code of Conduct.
+
+A public record allows the community to monitor the enforcement of the Code of
+Conduct and propose changes to the reporting and enforcement process if
+necessary.
+
+## Code of Conduct Reports and Responses
+
+Updated as of June 30, 2022
```

Comment:
Is there a deliberate reason not to make it today?

Reply:
Could say here: "Updated as of <date>. Reports listed in reverse chronological order."

Reply:
I like adding the note on the order. We decided that since we plan to commit to twice a year updates, making it June 30 let's us do it on Dec 30, and June 30 going forward.

Reply:
That actually makes a lot of sense. But I'd expect others (particularly those involved with future issues) might want to know this, so can I suggest a statement is put at the top of the log?